presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ScalarImplementation.java

```diff
@@ -419,15 +421,15 @@ else if (parameterType.isPrimitive()) {
 
                     boolean currentNullFlag = false;
                     if (i + 1 < method.getParameterCount()) {
-                        boolean foundIsNull = false;
                         annotations = method.getParameterAnnotations()[i + 1];
-                        for (Annotation annotation : annotations) {
-                            if (annotation instanceof IsNull) {
-                                foundIsNull = true;
-                                break;
+                        if (Stream.of(annotations).anyMatch(IsNull.class::isInstance)) {
+                            Class<?> isNullType = method.getParameterTypes()[i + 1];
```

Comment:
Validate that this doesn't contain `@SqlType`, `@SqlNullable` or other meta annotations